### DISCUSSION TITLE
q-page-container lost inline style  padding

### DISCUSSION THREAD

**Discussion by wyzpl** [quasarframework/quasar#14891] [CLI - Capacitor / Cordova modes]
use vue-router  push method forward multilevel page。Then use back method ，q-page-container lost inline style  padding

**Comment by pdanpdan** [comment]
can you make a reproduction, please?

**Comment by wyzpl** [comment]
I found the reason， Thank you




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Popescu ***@***.***&gt;; 
发送时间: 2022年11月14日(星期一) 下午5:58
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [quasarframework/quasar] q-page-container lost inline style  padding (Discussion #14891)






请你复制一张好吗？

—
直接回复此邮件，在GitHub上查看，或取消订阅.
***@***.***与&gt;.
